### PR TITLE
Supporting style

### LINKED ISSUE #5270: How to keep table element attributes after launching bootstraptable?
Hi, I'm encountering a small issue referring table element attributes. It's simple like this:

[html file]
`<thead> ... </thead>`
`<tbody>`
`...`
`<td style="background-color:red">...</td>`
`...`
`</tbody>`
`</table>`

[js file]
`$("#mytable").bootstrapTable();`

This js script will actually remove the background color of that td, and I'd like to keep it.

Any help would be appreciated!

### PR DESCRIPTION
**Pull request by djhvscf**
**Bug fix?**
No

**New Feature?**
Yes

**Resolve an issue?**
Fix #5270

**Example(s)?**
https://live.bootstrap-table.com/code/djhvscf/4561

### CODE REVIEW

File: src/bootstrap-table.js [hunk 1]
```diff
@@ -1336,6 +1340,7 @@ class BootstrapTable {
       Utils.sprintf(' %s', htmlAttributes.length ? htmlAttributes.join(' ') : undefined),
       Utils.sprintf(' id="%s"', Array.isArray(item) ? undefined : item._id),
       Utils.sprintf(' class="%s"', style.classes || (Array.isArray(item) ? undefined : item._class)),
+      Utils.sprintf(' style="%s"', Array.isArray(item) ? undefined : item._style),
```
Comment (UtechtDustin) [inline src/bootstrap-table.js:1343]: Im not sure what happens if you apply `undefined` on the `sprintf` function.
Could you check that ?

**Review by UtechtDustin — CHANGES_REQUESTED** [review]

File: src/bootstrap-table.js (hunk 1 above)
Comment (djhvscf) [reply src/bootstrap-table.js:1343]: It returns empty when the parameters are undefined https://jsfiddle.net/djhvscf/xv0f6cyd/

Comment (UtechtDustin) [reply src/bootstrap-table.js:1343]: So it would make more sense to check it with a if to prevent empty style attributes (`style=""`).

Comment (wenzhixin) [reply src/bootstrap-table.js:1343]: It's OK: https://github.com/wenzhixin/bootstrap-table/blob/develop/src/utils/index.js#L2

**Review by wenzhixin — APPROVED** [review]

File: src/bootstrap-table.js (hunk 1 above)
Comment (UtechtDustin) [reply src/bootstrap-table.js:1343]: oh okay, i thought %s would be replaced with an empty string :D

**Pull request merged**